src/create-template-orders.js

```diff
@@ -161,12 +160,18 @@ async function _createTemplateOrderAndPayments(checkoutOrder, orderDraft) {
     await apiRoot.orders().importOrder().post({ body: orderDraft }).execute()
     stats.createdTemplateOrders++
   } catch (err) {
-    if (!_isDuplicateOrderError(err)) {
-      const errMsg =
-        `Unexpected error on creating template order from checkout order with number: ${checkoutOrder.orderNumber}.` +
-        ` Line item: ${JSON.stringify(orderDraft.lineItems)}`
-      throw new VError(err, errMsg)
-    }
+    if (!_isDuplicateOrderError(err))
+      throw new VError(
+        {
+          cause: err,
+          info: {
+            orderNumber: checkoutOrder.orderNumber,
+            lineItems: orderDraft.lineItems,
```

Comment:
I thought that if the error happens (a potential NPE when some information is missing on the checkoutOrder before sending a request) then this might be informational, as there will no information other than orderNumber. But you are right about the duplicated information, but I would rather keep this for now.